albeanth-IdahoFalls.md

```diff
@@ -0,0 +1,10 @@
+# Idaho Falls, ID
+
+### Restaurants
+
+### Local Attractions
+
+- Hiking
+- Fishing
```

Comment:
I need to go get my 🐟  liscense. One of these days I'll learn how to spell liscense.